decidim-meetings/app/views/decidim/meetings/meetings/show.html.erb

```diff
@@ -0,0 +1,49 @@
+<% content_for(:title, translated_attribute(meeting.title)) %>
+
+<div class="row column view-header">
+  <h2 class="heading2"><%= translated_attribute meeting.title %></h2>
+  <div class="author-data">
+    <div class="author-data__main">
+      <div class="author author--inline">
+        <span class="author__avatar">
+          <%= image_tag meeting.author.avatar.url %>
+        </span>
+        <%= l meeting.created_at.to_date, format: :long %> <%= t(".by") %> <%= meeting.author.name %>
```

Comment:
That is, I don't know if it's needed or not, but since the layout had it I assumed we needed it. @josepjaume what should I do about this?